ui/oopsyraidsy/oopsyraidsy_config.js

```diff
@@ -222,6 +222,14 @@ UserConfig.registerOptions('oopsyraidsy', {
       type: 'checkbox',
       debugOnly: true,
     },
+    {
+      id: 'ClickToCopy',
+      name: {
+        en: 'Enable click-to-copy',
```

Comment:
can you added this line, thank you.
`fr: 'Activer le clic pour copier',`

Reply:
I just removed this option, so didn't apply this.